tck/src/main/java/org/eclipse/microprofile/lra/tck/TckTests.java

```diff
@@ -648,7 +647,14 @@ private void multiLevelNestedActivity(CompletionType how, int nestedCnt) throws
              * - C2 is completed and then compensated
              * - C3, ... are completed
              */
-            lraClient.cancelLRA(uris[1]); // compensate the first nested LRA
+
+            // compensate the first nested LRA in the enlisted resource
+            tckSuiteTarget
```

Comment:
This is different then what I always had in mind when we discussed nested LRAs.

Because here we explicitly call an endpoint which was used as a nested call.

What I had in mind (and what I though we were discussing) is that @Compensate can be called for a nested LRA although @Complete is already called.

'Parent LRA' 
    -> nested1 -> success -> @Complete for C1.
    -> nested2 -> success -> @Complete for C2.
    -> nested3 -> Failure
  -> @Compensate for C1 and C2 need to be called.

Reply:
This is correct, but this test tests different scenario. Namely, that the **user** can cancel already closed nested LRA.

Reply:
Ah, I see now that I misunderstood your comment. That is indeed the current scenario.